hackathon_site/dashboard/frontend/src/components/teamDetail/TeamCheckoutOrderTable/TeamCheckoutOrderTable.tsx

```diff
@@ -0,0 +1,227 @@
+                            <Typography variant="h2">{`Order #${checkoutOrder.id}`}</Typography>
+                        </Grid>
+                        <br />
+                        <TableContainer component={Paper} elevation={2} square={true}>
+                            <Table className={styles.table} size="small">
+                                <TableHead>
+                                    <TableRow>
+                                        <TableCell className={styles.widthFixed} />
+                                        <TableCell className={styles.width6}>
+                                            Name
+                                        </TableCell>
+                                        <TableCell
+                                            className={`${styles.width1} ${styles.noWrap}`}
+                                        >
+                                            Info
+                                        </TableCell>
+                                        <TableCell
+                                            className={`${styles.width1} ${styles.noWrap}`}
+                                        >
+                                            Qty
+                                        </TableCell>
+                                        <TableCell
+                                            className={`${styles.width1} ${styles.noWrap}`}
+                                        >
+                                            Qty to return
+                                        </TableCell>
+                                        <TableCell
+                                            className={`${styles.width6} ${styles.noWrap}`}
+                                            align={"right"}
+                                        >
+                                            Qty remaining
+                                        </TableCell>
+                                        <TableCell
+                                            className={`${styles.width1} ${styles.noWrap}`}
+                                        >
+                                            Condition
+                                        </TableCell>
+                                        <TableCell>
+                                            <Checkbox
+                                                color="primary"
+                                                style={{
+                                                    marginLeft: "-15px",
+                                                }}
+                                            />
+                                        </TableCell>
+                                    </TableRow>
+                                </TableHead>
+                                <TableBody>
+                                    {checkoutOrder.request.map((row) => (
+                                        <TableRow key={row.id}>
```

Comment:
Here come with the logic error. As we can see in the mockData.js
<img width="602" alt="CleanShot 2022-08-04 at 17 06 27@2x" src="https://user-images.githubusercontent.com/36619969/182952693-5fc5c56c-4df3-4910-b985-7611b3b7e3ad.png">

The hardware is stored in a list, and the index for a list start from 0. But the hardware id start from 1. And the row.id you used here start from 1. So what should you do to make it logically correct?